frappe/website/doctype/blog_post/blog_post.py

```diff
@@ -211,3 +224,6 @@ def get_blog_list(doctype, txt=None, filters=None, limit_start=0, limit_page_len
 			post.avatar = "/" + post.avatar
 
 	return posts
+
+def getWords(content):
+	return re.compile('\w+').findall(content)
```

Comment:
You could use `len(content.split())` instead of regex and use it directly in the `set_read_time` function instead of a separate function.